x-pack/plugins/fleet/server/saved_objects/index.ts

```diff
@@ -162,6 +162,7 @@ const getSavedObjectTypes = (): { [key: string]: SavedObjectsType } => ({
           dynamic: false, // we aren't querying or aggregating over this data, so we don't need to specify any fields
           properties: {},
         },
+        allow_edit: { type: 'keyword', index: false },
```

Comment:
Any reason to specify the field in the mappings, given it's `index: false`?

Reply:
not really I will remove it 👍

Reply:
Look's like the type is need other I get a typescript error